skbio/util/_misc.py

```diff
@@ -226,7 +229,9 @@ def safe_md5(open_file, block_size=2 ** 20):
     return md5
 
 
-@experimental(as_of="0.4.0")
+@deprecated(as_of="0.4.2-dev", until="0.4.4",
```

Comment:
`until` is a little tricky here, our next release is *hopefully* going to be `0.5.0`, @jairideout how do you want to handle this?